package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "flux",
-  "version": "2.0.3",
+  "version": "2.1.0",
```

Comment:
Do this in a separate commit, not a part of this PR (better for tagging releases)